src/components/FilterMenu/FilterMenu.jsx

```diff
@@ -0,0 +1,191 @@
+import { StarRating } from "shoto-ui";
+import "./filterMenu.css";
+
+export const FilterMenu = ({ brands, dispatch, ratingFilter, priceFilter }) => {
+  const yellowColor = "rgb(255, 149, 41)";
+  return (
+    <div className="container-filterMenu">
+      {brands && (
+        <div className="filter-block">
+          <div className="filter-heading">Brands</div>
+          <div>
+            {brands.map((item) => (
+              <input type="checkbox" />
+            ))}
+          </div>
+        </div>
+      )}
+
+      <div className="filter-block">
+        <div className="filter-heading">
+          <div>Price</div>
+          {priceFilter && (
+            <div
+              className="filter-clear"
+              onClick={() =>
+                dispatch({
+                  type: "SET_PRICE_FILTER",
+                  payload: null,
+                })
+              }
+            >
+              {"<"} Clear
+            </div>
+          )}
+        </div>
+
+        <div

```

Comment:
This code can be extracted into a separate component. There seems to be unnecessary code repetition